scripts/CMakeLists.txt

```
@@ -4,3 +4,4 @@ configure_file(eosio-tn_down.sh eosio-tn_down.sh COPYONLY)
 configure_file(eosio-tn_roll.sh eosio-tn_roll.sh COPYONLY)
 configure_file(eosio-tn_up.sh eosio-tn_up.sh COPYONLY)
 configure_file(abi_is_json.py abi_is_json.py COPYONLY)
+configure_file(postinst postinst)
```

I wouldn't hate if `<output>` in [`configure_file()`](https://cmake.org/cmake/help/latest/command/configure_file.html) was `.` instead of a copy paste of `<input>`. But this is kind of an all or nothing thing for this file. No big deal if we don't do it.
```suggestion
configure_file(postinst .)
```

---

Addressed: [be154da](https://github.com/AntelopeIO/leap/pull/818/commits/be154da6036dbe810631b53b32afee58491767fc)

---

Thanks!